rudder-web/src/main/scala/com/normation/rudder/web/services/ReportDisplayer.scala

```diff
@@ -127,28 +132,117 @@ class ReportDisplayer(
     AnonFunc(ajaxCall)
   }
 
+
+  private[this] def displayIntro(report: NodeStatusReport): NodeSeq = {
+    def explainCompliance(info: RunAndConfigInfo): NodeSeq = {
+      val dateFormat = DateTimeFormat.forPattern("YYYY-MM-dd HH:mm:ss")
+
+      def currentConfigId(expectedConfigId: NodeConfigIdInfo) = {
+        s"Current configuration ID for the node is '${expectedConfigId.configId.value}' (generated on ${expectedConfigId.creation.toString(dateFormat)})"
+      }
+
+      info match {
+        case ComputeCompliance(lastRunDateTime, lastRunConfigId, expectedConfigId, expirationDateTime, missingReportStatus) =>
+          (
+            <p>{currentConfigId(expectedConfigId)}</p>
+            <p>Last complete agent runs  has correct configuration ID, started on node at {lastRunDateTime.toString(dateFormat)}</p>
+          )
+
+        case Pending(expectedConfigId, optLastRun, expirationDateTime, missingReportStatus) =>
+          val runInfo = optLastRun match {
+            case None => "No recent runs we received for that node"
+            case Some((date, id)) =>
+              s"Last run for that node executed at ${date.toString(dateFormat)} for configuration ID ${id.configId.value} " +
```

Comment:
What does "executed" mean in this context? The word should definitely be changed, but since I don't know what it refers to, it's hard to say what to.

If it's when the previous run happened on a node, use "started".